CalculateTests/CalculatorTests.cs

```diff
@@ -0,0 +1,118 @@
+﻿
+
+using Calculate;
+
+namespace CalculateTests;
+
+public class CalculatorTests
+{
+    [Theory]
+    [InlineData(1,2,3)]
+    [InlineData(2,-3,-1)]
+    public void Add_TwoDoubles_SuccessfulAdd(int a, int b, double result)
+    {
+        // Act
+        double number = Calculator.Add(a, b);
+
+        //Assert
+        Assert.Equal(result, number);
+
+    }
+
+    [Theory]
+    [InlineData('+', 1,2,3)]
+    [InlineData('-', 2,3,-1)]
+    [InlineData('*', 2,3,6)]
+    [InlineData('/', 6,3,2)]
+    public void MathematicalOperations_UsingGenericDictionary_SuccessfulAdd(char operationSymbol, int a, int b, double result)
+    {
+        //Arrange
+        var operation = Calculator.MathematicalOperations[operationSymbol];
+
+        //Act
+        double number = operation(a, b);
+
+        //Assert
+        Assert.Equal(result, number);
+    }
+
+    [Theory]
+    [InlineData("1 + 2", 3)]
+    [InlineData("2 - 3", -1)]
+    [InlineData("2 * 3", 6)]
+    [InlineData("6 / 3", 2)]
+    public void TryCalculate_ValidInputs_SuccessfulOperation(string input, double result)
+    {
+        //Act
+        bool canCalculate = Calculator.TryCalculate(input, out double output);
+
+        //Assert
+        Assert.True(canCalculate);
+        Assert.Equal(result, output);
+    }
+
+    [Theory]
+    [InlineData("1 2")]
+    [InlineData("1 2 3")]
+    [InlineData("1 + 2 3")]
+    [InlineData("a * 3")]
+    [InlineData("1 ++ 2")]
```

Comment:
Can it calculate if the inline data has an extra white space?
```suggestion
    [InlineData("1 +  2")]
```

Reply:
Good question, that'd be good to check.